recipes/qpdf/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+        return {
+            "gcc": "5",
+            "Visual Studio": "19.10",
+            "clang": "3.4",
+            "apple-clang": "10",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        # https://qpdf.readthedocs.io/en/stable/installation.html#basic-dependencies
+        if self.options.with_crypto == "openssl":
+            self.requires("openssl/3.0.5")
+        self.requires("zlib/1.2.12")
+        self.requires("libjpeg-turbo/2.1.4")
+
+
+    def validate(self):
+        # validate the minimum cpp standard supported. For C++ projects only
+        if self.info.settings.compiler.cppstd:
+            check_min_cppstd(self, self._minimum_cpp_standard)
+        minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
+        if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._minimum_cpp_standard}, which your compiler does not support.")
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.24.1")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+                  destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_SHARED_LIBS"] = self.options.shared
+        tc.variables["BUILD_STATIC_LIBS"] = not self.options.shared
+        # https://qpdf.readthedocs.io/en/latest/installation.html#build-time-crypto-selection
+        tc.variables["USE_IMPLICIT_CRYPTO"] = False
+        if self.options.with_crypto == "native":
+            tc.variables["REQUIRE_CRYPTO_NATIVE"] = True
+            tc.variables["REQUIRE_CRYPTO_GNUTLS"] = False
+            tc.variables["REQUIRE_CRYPTO_OPENSSL"] = False
+        elif self.options.with_crypto == "openssl":
+            tc.variables["REQUIRE_CRYPTO_NATIVE"] = False
+            tc.variables["REQUIRE_CRYPTO_GNUTLS"] = False
+            tc.variables["REQUIRE_CRYPTO_OPENSSL"] = True
+
+        tc.generate()
+        tc = CMakeDeps(self) # init requirements from dependencies
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE.txt", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+
+        # some files extensions and folders are not allowed. Please, read the FAQs to get informed.
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
Are you sure that project is generating all those files? .la, .pdb, .pc? Please, remove what's not used, check the build log after passing by the CI.